clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/ops/UpsertEntityTagsAtomicOperation.java

```diff
@@ -93,6 +83,83 @@ public Void operate(List priorOutputs) {
     return null;
   }
 
+  @VisibleForTesting
+  public UpsertEntityTagsDescription setIdAndIdPattern(UpsertEntityTagsDescription description) {
```

Comment:
If it were me, I'd probably just leave this private and static (_if possible_). 

The groovy test will be able to access it no problem.